halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/ui/v1/Ui.java

```diff
@@ -56,6 +52,10 @@ public void failure(String message) {
     reset(color.BOLD + color.RED + "! " + color.RESET + message);
   }
 
+  public void remediation(String message) {
```

Comment:
Just a thought. If you have a non-interactive mode where you inspect a configuration (e.g. post deploy problems when perhaps asynchronous changes were made that invalidated the configuration that was once good) then it would be useful to accumulate these so you could summarize all the recommendations at the end. That could be true of problems and other concerns as well. Usually that's done with local state but having these specialized reporting methods suggests there might be a way to bake that into the model [later].